index.html

```diff
@@ -288,7 +289,7 @@ <h2 class="categorie-title">
             <h1 class="footer-title">
                 I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
             </h1>
-            <form action="https://formspree.io/f/mrgjzokg" method="post">
+            <form action="https://formspree.io/f/mrgjzokg" method="post" class="form-input">
                 <ul class="form-fields">
                     <li class="input-name">
                         <input type="text" name="name" id="name" maxlength="30" placeholder="Full name" required>
```

Comment:
While you made a good job implementing the features, however, most of them are not addressed yet. Let's fix them before moving to the next task.
- [ ] Kindly ensure that an error is displayed when the name and email fields are blank and the form is not submitted as per the requirements
- [ ] email field containing uppercase characters should generate an error as per the project requirements. You can check if the email is all lowercase by comparing if the email and it's lowercase converted version is the same, ie: `email === email.toLowerCase()`, utilizing regular expressions, or looping through each letter of the email string.
- [ ] check the screenshot
![Screenshot from 2021-12-17 01-36-29](https://user-images.githubusercontent.com/20245164/146459347-e325f333-bb9c-480e-8636-a608f7315574.png)